### ISSUE TITLE
Question on the logpt_neg param

### ISSUE THREAD

**Issue by taekwan-lee** [slryou41/AnchorLoss#2]
I read your paper on anchor loss with curiosity.
Right now I'm trying to implement the code using tensorflow, and I came up with some questions.

1) For `logpt_neg`, you calculated `F.logsigmoid(1-input)`, but as far as I understand, if the `input` parameter is non-sigmoid logits, then it should be `torch.log(1-sig(input))`, isn't it?

2) Did you experience any 'NaN' loss problem during training? if you did, what was your solution for the problem?

**Comment by slryou41** [comment]
1. Thank you so much for the correction! We have updated the code.
2. We didn't experience 'NaN' with the current version of the code, but when we used log operation on top of the output directly (not using the functions from PyTorch e.g. ``F.logsigmoid``), it sometimes generated 'NaN'. We added an epsilon term (1e-9) before applying logarithm and it helped but I'd recommend using the internal functions.
